### ISSUE TITLE
DataTexture + logarithmicDepthBuffer has some artifacts

### ISSUE THREAD

**Issue by Amoki** [xeokit/xeokit-sdk#1145]
**Describe the bug**
When using `DataTexture` and `logarithmicDepthBuffer`, there are some glitches on objects with custom rendering:
![image](https://github.com/xeokit/xeokit-sdk/assets/1349751/b4a551b7-007d-4765-ae3e-15336377e6f1)
![image](https://github.com/xeokit/xeokit-sdk/assets/1349751/3b67172c-a8c3-4a54-8dc2-d624c1f2c56c)

I user default xeokit window rendering and this custom rendering for spaces:
```javascript
new xeokitSdk.XKTLoaderPlugin(viewer, {
  objectDefaults: {
    IfcSpace: {
      pickable: false,
      opacity: 0.2,
      colorize: [0, 0, 1],
    },
  },
});
```

cc @tmarti

**Comment by xeolabs** [comment]
I had a go at this issue a few days back, but couldn't figure it out, hence making log depth buf disabled by default.

Notes: 

* this is happening when edges are made visible in models (while log depth buf enabled)
* seems to happen with models having large vertex coordinates

I think it's that edges and triangles somehow end up with very different depth values, despite using the same shader logic for log depth buf. I tried to resolve this one, but couldn't find why they had such different depths. 

Originally I had made log depth buf enabled by default.

Because of this issue, and to make section plane controls align with surfaces properly, made log depth buf disabled by default.
